deploy/vagrant/vbox/buildbox.sh

```diff
@@ -12,9 +14,16 @@ if [ -f tachyon-dev.box ]; then
   rm -f tachyon-dev.box
 fi
 
-vagrant up
-vagrant package --output tachyon-dev.box default
-vagrant destroy -f
+BASE_BOX='chef/centos-7.0'
+AVAIL_BOX=$(vagrant box list | grep $BASE_BOX | grep '(virtualbox, 1.0.0)')
+
```

Comment:
@yuzhu I just had another thought and updated my earlier comments which seems happened at the same time as you wrote yours. Please read my comments again and let me know your thoughts. 

I think the better fix as I suggested, is to use "create" script to create tachyon-dev.box where all tools, and tachyon, ufs are already downloaded. That will be much faster. What do you think?